glustercli/cmd/snapshot-info.go

```diff
@@ -47,6 +47,7 @@ func snapshotInfoDisplay(snap api.SnapGetResp) {
 	fmt.Println("Snapshot Volume ID:", vol.ID)
```

Comment:
Commit message, PR title and PR description can be improved. Here's an example: 

`volume: add capacity field`

```
Added capacity field to volume GET AP response.
Added capacity field to volume GET AP response. This is currently
returned only for volumes provisioned dynamically and their volume
snapshots.

The CLI prints the volume capacity in a human friendly format.
```

Reply:
make more sense. done